<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DorsalesTagsDialog</class>
 <widget class="QDialog" name="DorsalesTagsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignaciones de tarjetas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/dorsal_icon.png</normaloff>images/dorsal_icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dorsal:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dorsalText">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CÃ³digo tarjeta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tagText"/>
       </item>
       <item>
        <widget class="QPushButton" name="leerTagButton">
         <property name="text">
          <string>Leer tarjeta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="asignarButton">
         <property name="text">
          <string>Asignar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Desasignar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="guardarButton">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cerrarButton">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
